tools/test-install/prepare-base

```diff
@@ -60,8 +60,8 @@ run apt-get install -y --no-install-recommends \
     "${extra_packages[@]}"
 
 run ln -sf /usr/share/zoneinfo/Etc/UTC /etc/localtime
-run locale-gen en_US.UTF-8 || true
-echo "LC_ALL=en_US.UTF-8" | run tee /etc/default/locale
+run locale-gen C.UTF-8 || true
+echo "LC_ALL=C.UTF-8" | run tee /etc/default/locale
```

Comment:
We may be able to in follow-up work remove explicit logic to set locale in our testing Dockerfiles; since the theory behind this set of changes is we won't need to set these explicitly except in our own scripts.